projects/storefrontlib/cms-components/myaccount/new-notification-preference/new-notification-preference.component.html

```diff
@@ -0,0 +1,45 @@
+<ng-container *ngIf="preferences$ | async as preferences">
+  <div *ngIf="preferences.length > 0; else loading">
+    <div role="status" [attr.aria-label]="'common.loaded' | cxTranslate"></div>
+    <div class="row d-flex justify-content-center">
```

Comment:
I think we should steer clear of bootstrap classes. If we ever want to drop bootstrap as a dependency, it will become more difficult if we use it with new features and create more tech debt. When creating new features is the easiest time to avoid these.

Reply:
modified as suggestion.